docs/routers/nokia-G240WB.md

```diff
@@ -0,0 +1,104 @@
+This guide was developed using a `Nokia G-240W-B` router. It should work for other models in the `G` series.
+
+### Device information
```

Comment:
This whole 'Device information' section can go. There is a lot of redundant and/or unnecessary information.

Reply:
Got it, make sense if removing dup info.